dynamodb_utils/awslambda.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import
+
+import importlib
+import json
+import logging
+import os
+
+from pynamodb.connection import Connection
+
+from . import dumper
+from .multi import awslambda
+
+logger = logging.getLogger(__name__)
+
+def handler(event, context):
+    if not isinstance(event, dict):
+        logger.error('event not a dict')
+        return
```

Comment:
i think this is kind of usless error handling, better to let it blow up when `event.get` blows up